pkg/cli/host.go

```diff
@@ -50,7 +50,7 @@ func newHostCommand(cfgFlags *configFlags) *cobra.Command {
 	}
 	create.Flags().StringVar(&createFlags.MachineType, gcpMachineTypeFlag, "n1-standard-4",
 		"Indicates the machine type")
-	create.Flags().StringVar(&createFlags.MinCPUPlatform, gcpMinCPUPlatformFlag, "Intel Haswell",
```

Comment:
I assume that this not configured in the gce api request if the value is not set, correct?

Reply:
correct, if set as empty it's handled as not set.